specification/license/post_start_trial/StartTrialLicenseRequest.ts

```diff
@@ -35,5 +36,15 @@ export interface Request extends RequestBase {
   query_parameters: {
     acknowledge?: boolean
     type_query_string?: string
+    /**
+     * Period to wait for a connection to the master node.
+     * @server_default 30s
+     */
+    master_timeout?: Duration
+    /**
+     * Period to wait for a response. If no response is received before the timeout expires, the request fails and returns an error.
+     * @server_default 30s
+     */
+    timeout?: Duration
```

Comment:
Only `master_timeout` seems supported, not `timeout`: https://github.com/elastic/elasticsearch/blob/ce7a0c86fb975f76dfbea5eff1cf51602dbed184/x-pack/plugin/core/src/main/java/org/elasticsearch/license/RestPostStartTrialLicense.java#L35